tests/kani/Makefile

```diff
@@ -0,0 +1,38 @@
+SHELL=/bin/bash
+
+# Assume cbmc and cbmc-viewer are installed in PATH
+#   skip ./kani/scripts/setup/ubuntu/install_cbmc.sh
+#   skip ./kani/scripts/setup/install_viewer.sh 2.11
+
+KANI=kani
+
+# ARCH is ubuntu or macos-10.15
+ARCH=ubuntu
+
+clone:
+	git clone https://github.com/model-checking/kani.git $(KANI)
+	cd $(KANI) && git submodule update --init
+
+# MacOS warning: If brew fails trying to install ctags over
+# universal-ctags, then replace ctags with universal-ctags
+# ./kani/scripts/setup/macos-10.15/install_deps.sh
+
+build:
+	PATH=$$HOME/.cargo/bin:$$PATH && cd $(KANI) && \
```

Comment:
This style makes it difficult to see which of these commands failed. Suggest:
```
build:
	PATH=$$HOME/.cargo/bin:$$PATH  $(MAKE) -C $(KANI) _build

_build:
	./scripts/setup/$(ARCH)/install_deps.sh
	./scripts/setup/install_rustup.sh
	cargo build --workspace
```

Reply:
Done something similar.  The `-C $(KANI)` won't work because it requires a second Makefile in the `$(KANI)` directory.